api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -114,6 +110,14 @@
         return returnStatusForJobCreation(job);
     }
 
+    private void checkIfCurrentUserCanAddJob() {
+        if (!jobService.checkIfCurrentUserCanAddJob()) {
+            String errorMsg = "User already has enough active or submitted jobs for their limit";
```

Comment:
This message will be presented to the users, I assume. Let's word it a bit differently so that it has more clarity: `You already have active export requests in progress. Please wait until they complete before submitting a new one.`